### ISSUE TITLE
Flow information missing in Historical Flows

### ISSUE THREAD

**Issue by martinscheu** [ntop/ntopng#8514]
**What would you like to add or change?**:
From the Flow Overview it would be helpful to have following data in historical flow as well:
- Client MAC address (somehow the server MAC address is there)
- Blacklist information, e.g. IPsum Threat Intelligence Feed
- Confidence of Application detection, e.g. DPI
- CommunityID
- HASSH from client and server

**Why do you and others need this?**:

**Comment by lucaderi** [comment]
Implemented with packet interfaace

**Comment by lucaderi** [comment]
For sure
- Client MAC address (somehow the server MAC address is there)
- Blacklist information, e.g. IPsum Threat Intelligence Feed
- Confidence of Application detection, e.g. DPI

are present with nProbe.

HASSH is not present (it needs to be exported). CommunityId is computed by Flow::getCommunityId(). It might not be correct with ICMP flows (TODO) but it is correct for all the other flows.

In summary
- TODO nprobe: export HASSH and ICMP info
- all the rest should be ok to display on ntopng
